partiql-ast/src/main/kotlin/org/partiql/ast/sql/Sql.kt

```diff
@@ -1,15 +1,38 @@
 package org.partiql.ast.sql
 
 import org.partiql.ast.AstNode
+import org.partiql.ast.sql.internal.InternalSqlDialect
+import org.partiql.ast.sql.internal.InternalSqlLayout
+
+/**
+ * No argument uses optimized internal. Leaving older ones for backwards-compatibility.
+ */
+public fun AstNode.sql(): String {
```

Comment:
Are we marking any of the existing pretty-printing APIs as deprecated and to be removed in a future major version?